### PR TITLE
Update the manual-setup.md instructions

### PR DESCRIPTION
**Pull request by jose8a**
Add a simple astro configuration file to the instructions to resolve development errors that show up without including this file.

### CODE REVIEW

**Comment by sarah11918** [comment]
Thanks for this, I think it's helpful!

Would you be willing to:
1. update the new Section 6 so that the project directory
b) includes astro.config.mjs 
a) looks more like it does here, with the branches instead of just dashes: https://docs.astro.build/en/core-concepts/project-structure/ 

2. In the new Section 5, include a link to https://docs.astro.build/en/core-concepts/framework-components/#customize-your-frameworks with a short line stating that if you want to use framework components in your project, this configuration file should specify any/all renderers.

File: src/pages/en/guides/manual-setup.md [hunk 1]
```diff
@@ -82,7 +82,22 @@ User-agent: *
 Allow: /
 ```
 
-## 5. Next steps
+## 5. Add a basic astro.config.mjs file into the project root
+
+Astro expects at least a minimal configuration file with the following minimal configuration content:
```
Comment (sarah11918) [inline src/pages/en/guides/manual-setup.md:87]: Astro is configured using `astro.config.mjs` at the root of the project. This file is optional if you do not need to configure Astro, but you may wish to create it now. If you want to include framework components such as React, Svelte, etc. in your project, here is where you will [manually add any necessary renderers](en/core-concepts/framework-components/#customize-your-frameworks).

(Something like that?)

**Comment by Siilwyn** [comment]
👋, if https://github.com/withastro/docs/pull/211 is okay maybe we can collaborate on this PR? Fits perfectly with the addition of the astro config.

**Comment by sarah11918** [comment]
> 👋, if #211 is okay maybe we can collaborate on this PR? Fits perfectly with the addition of the astro config.

👋 I commented there in #211 ! That PR is changing different files than the one file here, so I think they're good to handle separately. Especially, this PR requires you to create from scratch an `astro.config.mjs` file, so what's in there by default doesn't matter. You get to write it all out by hand!  😄 

I think the *other* PR requires more effort, (and clarification from engineering what they want to document!) because in those files, we have to deal with documenting what you get by default. Can't get anything by default if you have to create the file yourself!

**Comment by sarah11918** [comment]
Hi @jose8a , thank you again for this contribution to the docs! I think it's a great idea to have this step, and I'm merging it after adding some surrounding information/instructions, and updating the project directory in Step 6 to include this added file.  :)

**Pull request merged**